src/components/shared/FilterNavigatorBar.js

```diff
@@ -29,34 +33,51 @@ class FilterNavigatorBar extends PureComponent<Props> {
   }
 
   render() {
-    const { className, items, selectedItem } = this.props;
+    const { className, items, selectedItem, profileSelection } = this.props;
     return (
-      <TransitionGroup
-        component="ol"
-        className={classNames('filterNavigatorBar', className)}
-      >
-        {items.map((item, i) => (
-          <CSSTransition
-            key={i}
-            classNames="filterNavigatorBarTransition"
-            timeout={300}
-          >
-            <li
-              data-index={i}
-              className={classNames('filterNavigatorBarItem', {
-                filterNavigatorBarRootItem: i === 0,
-                filterNavigatorBarBeforeSelectedItem: i === selectedItem - 1,
-                filterNavigatorBarSelectedItem: i === selectedItem,
-                filterNavigatorBarLeafItem: i === items.length - 1,
-              })}
-              title={item}
-              onClick={this._onLiClick}
+      <div className={classNames('filterNavigatorBar', className)}>
+        <TransitionGroup
+          component="ol"
+          className={classNames('filterNavigatorBar', className)}
+        >
+          {items.map((item, i) => (
+            <CSSTransition
+              key={i}
+              classNames="filterNavigatorBarTransition"
+              timeout={300}
             >
-              <span className="filterNavigatorBarItemContent">{item}</span>
-            </li>
-          </CSSTransition>
-        ))}
-      </TransitionGroup>
+              <li
+                data-index={i}
+                className={classNames('filterNavigatorBarItem', {
+                  filterNavigatorBarRootItem: i === 0,
+                  filterNavigatorBarBeforeSelectedItem: i === selectedItem - 1,
+                  filterNavigatorBarSelectedItem: i === selectedItem,
+                  filterNavigatorBarLeafItem: i === items.length - 1,
+                })}
+                title={item}
+                onClick={this._onLiClick}
+              >
+                <span className="filterNavigatorBarItemContent">{item}</span>
+              </li>
+            </CSSTransition>
+          ))}
+        </TransitionGroup>
+        {profileSelection.hasSelection &&
+        className === 'profileFilterNavigator' ? (
```

Comment:
We don't need to check the `className`, just the existence of `uncommitedItem`. So that we don't have an hardcoded check :)